src/Veldrid.VirtualReality/Veldrid.VirtualReality.csproj

```diff
@@ -31,6 +31,18 @@
     </Content>
   </ItemGroup>
 
+
+  <ItemGroup>
+    <Folder Include="native\win-x86\" />
+  </ItemGroup>
+
+
+  <ItemGroup>
+    <None Update="native\win-x86\openvr_api.dll">
```

Comment:
The x64 binary is included and packaged a few lines above, I think the x86 DLL should follow the same pattern. Currently, it won't be included in the NuGet package.